pkgs/racket-doc/scribblings/reference/treelists.scrbl

```diff
@@ -317,6 +317,20 @@ results. For a constant-time @racket[proc], this operation takes
 (treelist-for-each items println)
 ]}
 
+@defproc[(treelist-filter [keep (any/c . -> . any/c)] [tl treelist?])
+         treelist?]{
+
+Produces a treelist with only members of @racket[tl] that satisfy
+@racket[keep].
+
+@examples[
+#:eval the-eval
+(treelist-filter even? (treelist 1 2 3 2 4 5 2))
+(treelist-filter odd? (treelist 1 2 3 2 4 5 2))
+(treelist-filter (λ (x) (not (even? x))) (treelist 1 2 3 2 4 5 2))
+(treelist-filter (λ (x) (not (odd? x))) (treelist 1 2 3 2 4 5 2))
+]}
```

Comment:
Needs a `history` note (and the same for the other additions)

Reply:
I've added history notes for 8.15.0.6, the current version. Is that okay, or should this include a version bump to 8.15.0.7?

Reply:
I think "8.15.0.6" is good.